pkg/tfgen/test_data/regress-611-schema.json

```diff
@@ -2291,8 +2291,7 @@
           "type": "string"
         },
         "forceDestroy": {
-          "type": "boolean",
-          "description": "Delete user even if it has non-Terraform-managed IAM access keys, login profile or MFA devices\n"
```

Comment:
Does this fix https://github.com/pulumi/pulumi-gcp/issues/1945?

Reply:
It does not fix it, but it reduces the number of incidents.

Reply:
Hi, for us this change removed half of the docstrings. Seems to be all multiline docstrings. Why are they getting removed with these changes?

Seems like a bug to me?

Reply:
Hmm, seems like its removing all docstrings that contain "Terraform" :face_exhaling:. Why is it not replacing them with "Pulumi" like in other places?

Reply:
@skirsten In which provider are you seeing this behaviour?

Reply:
Doesn't matter, it affects all providers (even the integrated testproviders): https://github.com/pulumi/pulumi-terraform-bridge/issues/2387

You can see all the deleted docstrings here: https://github.com/genesiscloud/pulumi-genesiscloud/pull/21/files

Reply:
hi @skirsten - we're sorry you're encountering this. 

Docsgen can be rather unintuitive. We don't actually as a rule transform any `terraform` string to `pulumi`, since it is impossible to tell if the text transformed would apply in a Pulumi context at all. As a precaution, we drop such sections from the docs entirely, although we are discussing potential refinements to this policy. Since your snippets weren't processed before this change, this drop did not occur.

I see you're maintaining the [pulumi-genesiscloud provider](https://github.com/genesiscloud/pulumi-genesiscloud). to avoid your docs loss, I'd recommend trying a docs edit rule during schemagen here.

The `ProviderInfo` has a `DocRules` [setting](https://github.com/pulumi/pulumi-terraform-bridge/blob/44154366585d48fe76e597463387b716896ccb83/pkg/tfbridge/info/info.go#L201) that will apply certain docs transformations at schema gen time.
This provider](https://github.com/pulumi/pulumi-gcp/blob/master/provider/doc_edits.go) has several examples for how to implement such transformations.
This would allow you to change your `Terraform` string to `Pulumi` before transformations apply, and you'd regain docs.
Please feel free to file a new issue if you're encountering further trouble.